docs/en/observability/synthetics-params-secrets.asciidoc

```diff
@@ -52,23 +30,42 @@ The values in the configuration file are read in the following order:
   When running a script using the CLI, {kib}-defined _Global parameters_ have no impact
   on the test because it won't have access to {kib}.
 
+[discrete]
+[[synthetics-params-secrets-define]]
+= Define params
+
+There are three options for defining params to be used in synthetic monitors.
```

Comment:
TBH, I'm not sure this intro adds much value. You've already introduced these 3 options in the previous section. I can see why you want the heading "Define params" to show up here, though. I suggest moving lines 11-31 to appear under this section so that you have a meaningful intro here. Maybe add links from your bulleted list above to each section.